src/jarabe/model/friends.py

```diff
@@ -156,9 +156,10 @@ def save(self):
         cp = ConfigParser()
 
         for friend in self:
-            section = friend.get_key()
-            cp.add_section(section)
-            cp.set(section, 'nick', friend.get_nick())
+            section = str(friend.get_key())
+            if section not in cp.sections():
+                cp.add_section(section)
+                cp.set(section, 'nick', friend.get_nick())
```

Comment:
Should L#162 be outside the if conditional code block?

Reply:
If section is already present , no need to set it again according to me.